tests/tpch/test_correctness.py

```diff
@@ -1,50 +1,63 @@
 import os
-import pathlib
 
 import coiled
 import dask
 import pandas as pd
 import pytest
 from distributed import LocalCluster, performance_report
 
-from .generate_answers import generate as generate_answers
-from .utils import get_dataset_path
-
-VERIFICATION_SCALE = 1
+from .utils import get_answers_path, get_cluster_spec, get_dataset_path
 
 pytestmark = pytest.mark.tpch_correctness
 
 
-@pytest.fixture(scope="module")
-def answers_path(tmp_path_factory):
-    path = tmp_path_factory.mktemp("answers")
-    return generate_answers(base_path=path)
+@pytest.fixture(params=[1, 10, 100], scope="session")
+def scale(request):
+    scale = request.param
+    if scale != 100:
+        pytest.skip(reason="Don't test everything by default")
+    return scale
```

Comment:
Note that this test doesn't adhere to `--scale X` because we likely want to test correctness at a smaller scale most of the time than the actual benchmarks we run.